src/xdp/trn_transit_xdp.c

```diff
@@ -729,6 +729,22 @@ static __inline int trn_process_geneve(struct transit_packet *pkt)
 		return XDP_ABORTED;
 	}
 
+	pkt->pod_label_value_opt = (void *)pkt->scaled_ep_opt + sizeof(*pkt->scaled_ep_opt);
```

Comment:
How do you know this points to pod label? If the pod label was not added on egress side, and only namespace label was added, you would be wrongly assuming that it is pod label isnt it?
Was this case really tested?

Reply:
On egress side, there will always pod label option and namespace label option be added, even when there are no label for the pod or namespace. When there is no label, the option value is 0.